internal/cfgplugins/policyforwarding.go

```diff
@@ -278,7 +296,13 @@ func InterfacePolicyForwardingConfig(t *testing.T, dut *ondatra.DUTDevice, a *at
 		switch dut.Vendor() {
 		case ondatra.ARISTA: // Currently supports Arista devices for CLI deviations.
 			// Format and apply the CLI command for traffic policy input.
-			helpers.GnmiCLIConfig(t, dut, fmt.Sprintf("interface %s.%d \n traffic-policy input tp_cloud_id_3_%d \n", aggID, a.Subinterface, a.Subinterface))
+			var cliConfig string
+			if params.Dynamic && a == nil && aggID == "" && params.AppliedPolicyName != "" && params.InterfaceID != "" {
+				cliConfig = fmt.Sprintf("interface %s \n traffic-policy input %s \n", params.InterfaceID, params.AppliedPolicyName)
```

Comment:
Using a multi-line string makes the template easier to read and maintain.


```suggestion
				cliConfig = fmt.Sprintf(`
				interface %s
					traffic-policy input %s
				!`, params.InterfaceID, params.AppliedPolicyName)
```